setup.py

```diff
@@ -47,7 +47,7 @@
         'drf-yasg',
         'django-click',
         'django-guardian',
-        'importlib_metadata',
+        'importlib_metadata>=3.6',
```

Comment:
If you are able to figure out a better method for specifying the tox deps instead of using `sitepackages=True`, can this change be undone?

Reply:
No, this change should stay regardless. It was incorrectly specified before. The change to allow `systemsitepackages` uncovered this error. The system had an older version of `importlib_metadata` so the old dependency was satisfied.  However, we use functionality only available in version 3.6 and later. The server would not start with the older version.